packages/stencil/src/components/datacom-tooltip/datacom-tooltip.css

```diff
@@ -33,7 +33,7 @@
   color: var(--text-color);
 
   /* TODO: var color */
```

Comment:
can you remove this TODO now that it's done

Reply:
Done